tests/mongodb.spec.js

```diff
@@ -0,0 +1,48 @@
+const should = require('should');
+const mongoose = require('mongoose');
+const dailyWeathers = require("../models/weatherSchema");
```

Comment:
I think the name of the model(class) should start with capital letter.
Check out the link below and consider changing it :)

Reference: [Mongoose Schemas v4.11.5](http://mongoosejs.com/docs/guide.html)